examples/basic/src/reducers/index.js

```diff
@@ -7,7 +7,7 @@ const user = (state = { isLoading: false }, action) => {
       return { ...state, isLoading: true };
 
     case FETCH_USER_FULFILLED:
-      return { ...state, isLoading: false, ...action.payload };
+      return { ...state, isLoading: false, ...action.payload, timestamp: new Date() };
 
```

Comment:
Calling new Date() here violates the convention that reducers should be pure functions.